docs/requirements-specification.md

```diff
@@ -57,4 +57,49 @@ _This content item details the use case overview of the system vision in its use
 
 ### UC5: Find expert in a field
 
-### UC6: Organizing capabilities
\ No newline at end of file
+### UC6: Organizing capabilities
+Only Create (C) use cases are explicitly defined here. Update (U) and Delete (D) need still to be defined and shall not contradict/invalidate Create use cases.
+
+| UC6a | Create capability group |
+|---|---|
+| Actors | Admin |If the capability group name exists already, the system prevents the Admin from creating the capability group. |
+| Pre-conditions | The admin is authenticated and has a valid, digital evidence. |
```

Comment:
Is evidence necessary to create a capability? What if you design a capability of which you do not have any evidence yet but wish to track it soon?

Reply:
Makes sense, evidence type is not a required information field anymore.

Reply:
But is evidence itself necessary? In other words, why should it not be possible to create a capability group when you do not currently have evidence for that capability group? If an administrator creates a capability group, they might not have any piece of evidence for it. They might just implement a recommendation from their research group. 

I would remove the second part of the precondition entirely.

Reply:
Ah, I completely misunderstood the precondition I partially copied from UC1. Brainfart. Will remove it here.